gogo/jline/src/main/java/org/apache/felix/gogo/jline/Posix.java

```diff
@@ -808,9 +809,26 @@ private void startShell(CommandSession session, Terminal terminal) {
         new Thread(() -> runShell(session, terminal), terminal.getName() + " shell").start();
     }
 
+    /**
```

Comment:
Do we really need this doc on a private messge whn al is written in commit message

Reply:
Agreed, see my previous comment